### ISSUE TITLE
Connection to Pravega

### ISSUE THREAD

**Issue by cheleb** [thatdot/quine#37]
[Pravega](https://pravega.io) is a alternative streaming storage engine form CNCF

**Comment by brackishman** [comment]
Can you provide context on the comment? Are you suggesting Quine could support integration with Pravega?

**Comment by cheleb** [comment]
Yes, I've started some experiment here https://github.com/cheleb/quine/tree/issues-37-pravega-connector
